solutions/standard/variables.tf

```diff
@@ -16,13 +16,13 @@ variable "use_existing_resource_group" {
 
 variable "resource_group_name" {
   type        = string
-  description = "The name of a new or an existing resource group in which to provision KMS resources to. If prefix input variable is passed then it will get prefixed infront of the value in the format of '<prefix>-value'."
+  description = "The name of a new or an existing resource group in which to provision key management resources to. If a prefix input variable is specified, it's added to the value in the `<prefix>-value` format."
 }
 
 variable "region" {
   type        = string
   default     = "us-south"
-  description = "The region in which to provision KMS resources. If using existing KMS, set this to the region in which it is provisioned in."
+  description = "The region in which to provision key management resources. If using an existing Key Protect or Hyper Protect Crypto Services key, set this to the region where it was provisioned."
```

Comment:
should be `If using an existing Key Protect or Hyper Protect Crypto Services instance`